packages/react/ds/src/text/text.tsx

```diff
@@ -11,7 +11,7 @@ function getTextClass({ as, size }: { as: TextAs; size?: TextSize }) {
     const sizes = {
       sm: 'xs:gi-text-sm md:gi-text-sm xl:gi-text-md',
       md: 'xs:gi-text-md md:gi-text-md xl:gi-text-lg',
-      lg: 'xs:gi-text-lg md:gi-text-lg xl:gi-text-xl',
+      lg: 'gi-text-lg',
```

Comment:
question: Should html components be updated as well?

Reply:
yes, you're right. I'm doing it